src/main/scripts/org/reaktivity/specification/nukleus/oauth/control/route/proxy/resolve.one.realm.with.extra.roles.then.route.proxy/controller.rpt

```diff
@@ -52,7 +56,20 @@ read 0x40000011
 read ${controlR05C.correlationId()}
 read ${authorization}
```

Comment:
This seems like it should be capturing `authorization` returned from nukleus instead, then also remove property.